introduction.tex

```diff
@@ -99,31 +99,40 @@ \subsection{Deployment Timeline}
 	\begin{center}
 		\includegraphics[scale=0.7]{./images/eu_future/power_plot.png}
 	\end{center}
-	\caption{Timeseries of installed nuclear capacity in \gls{EU}.}
+	\caption{The timeseries of installed nuclear capacity in the EU are separated by \texttt{Region}s in \Cyclus.
```

Comment:
why is s not in the \texttt{} brackets?

Reply:
It was my attempt of separating the cyclus language and normal language. I do see how it makes the words look a little weird

Reply:
ok, that makes sense. you can leave it if you like.